bin/travis-deploy-phar.sh

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env bash
 
+set -e
+
 git clone https://${GITHUB_TOKEN}@github.com/psalm/phar.git > /dev/null 2>&1
-cp build/psalm.phar build/psalm.phar.asc phar/
 cd phar
+rm -rf !(".git")
+cp ../build/psalm.phar ../build/psalm.phar.asc ../assets/psalm-phar/* .
+mv dot-gitignore .gitignore
 git config user.email "travis@travis-ci.org"
 git config user.name "Travis CI"
-git add psalm.phar psalm.phar.asc
+git add --all .
```

Comment:
This means that deleting a file from assets/psalm-phar will automatically delete it from the psalm/phar repo. The latter becomes purely a deployment medium.